wdio-a11y.conf.js

```diff
@@ -0,0 +1,308 @@
+/*require('ts-node').register({ transpileOnly: true });
+module.exports = require('./wdio.conf.ts');*/
+require('ts-node').register({ transpileOnly: true });
+exports.config = {
+    //
+    // ====================
+    // Runner Configuration
+    // ====================
+    //
+    // WebdriverIO allows it to run your tests in arbitrary locations (e.g. locally or
+    // on a remote machine).
+    runner: 'local',
+    //
+    // ==================
+    // Specify Test Files
+    // ==================
+    // Define which test specs should run. The pattern is relative to the directory
+    // from which `wdio` was called. Notice that, if you are calling `wdio` from an
+    // NPM script (see https://docs.npmjs.com/cli/run-script) then the current working
+    // directory is where your package.json resides, so `wdio` will be called from there.
+    //
+    specs: [
+        './e2e/wdio/**/*.acs-spec.ts'
+    ],
+    // Patterns to exclude.
+    exclude: [
+        './e2e/wdio/**/checkbox-group.e2e-spec.ts',
```

Comment:
do we need to exclude this from accessibility tests?

Reply:
Done.